src/main/services/schedule.ts

```diff
@@ -106,15 +40,26 @@ export async function startSchedule(
             runTasksDebounced(server!, piscina!, 'runEveryDay')
     })
 
+    const unlock = await startRedlock(
+        server,
+        LOCKED_RESOURCE,
+        () => {
+            weHaveTheLock = true
+            onLock?.()
+        },
+        () => {
+            weHaveTheLock = false
+        },
+        server.SCHEDULE_LOCK_TTL
+    )
```

Comment:
Fair point! I guess I didn't notice because of pycharm that shows the missing pieces:
![image](https://user-images.githubusercontent.com/53387/116579520-c08e7880-a912-11eb-9730-0542af09c337.png)

I changed the function to take on options object instead.